addons/mass_mailing/static/src/js/mass_mailing_html_field.js

```diff
@@ -82,52 +82,59 @@ export class MassMailingHtmlField extends HtmlField {
             return super.commitChanges();
         }
         if (!this._isDirty()) {
-            return;
+            // In case there is still a pending change while committing the
+            // changes from the save button, we need to wait for the previous
+            // operation to finish, otherwise the "inline field" of the mass
+            // mailing might not be saved.
+            return this._previousCommitChange;
         }
 
-        if (this.wysiwyg.$iframeBody.find('.o_basic_theme').length) {
-            this.wysiwyg.$iframeBody.find('*').css('font-family', '');
-        }
+        this._previousCommitChange = (async () => {
```

Comment:
I suggest `this._pendingCommitChanges`. Even if you strongly prefer to keep `previous` rather than `pending`, you forget the "s" at the end of `commitChanges`. Also, in the commit title you wrote `commitCange` and also forgot the "s" in the commit message.